django-nonrel/ocl/Dockerfile

```diff
@@ -1,7 +0,0 @@
-FROM python:2.7
-ENV PYTHONUNBUFFERED 1
-RUN mkdir /code
-WORKDIR /code
-ADD requirements.txt /code/
-RUN pip install -r requirements.txt
-ADD . /code/
```

Comment:
Why are you removing it? I understand you published a version of an image to docker hub, but how will we publish a new version of the image without this file?